app.js

```diff
@@ -1,3 +1,33 @@
+var express = require('express');
+var trucks = require('./trucks');
+
+var app = express();
+
+app.set('port', 3000);
+app.use(express.static('public'));
+
+app.get('/trucks', function(request, response) {
+	var allTrucks = trucks.getTrucks();
+	response.send(allTrucks);
+});
+
+app.get('/trucks/:trucks', function(request,response) {
+	var truckName = request.params.trucks;
+	response.send(trucks.getTruck(truckName)[0]);
```

Comment:
You'll want your getTruck method to return just an object so it doesn't have to be referenced by accessing it from an array.